provisioning_templates/snippet/http_proxy.erb

```diff
@@ -0,0 +1,12 @@
+<%#
```

Comment:
OK, let's pull it in for now, maybe we can even use it to generalize some of the proxy stuff later...